### PR TITLE
Finished refactoring.

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**
### Finished refactoring

  *I finished refactoring for all modals, images screen and removed unnecessary code.
Screenshot from 2023-06-07 17-58-58
Screenshot from 2023-06-07 17-59-54
Screenshot from 2023-06-07 18-00-46

### CODE REVIEW

**Comment by M7Arman** [comment]
@MiqayelChaloyan I see modals borders are different, please align them

File: components/Header/Header.jsx [hunk 1]
```diff
@@ -66,10 +67,14 @@ const Header = ({
...
+      <PermissionModal
         isModalVisible={isModalVisible}
-        setModalVisible={setModalVisible}
-        navigation={navigation}
+        children={
+          <LogoutModal
+            navigation={navigation}
+            setModalVisible={setModalVisible}
+          />
+        }
```
Comment (M7Arman) [inline components/Header/Header.jsx:77]: why not like this?
```
<PermissionModal>
  < LogoutModal />
</PermissionModal>
```

File: screens/ImagesScreen/index.js [hunk 2]
```diff
@@ -2,40 +2,48 @@ import PropTypes from 'prop-types';
...
       setImageUrl(result[0].url);
+      setModalVisible(false);
       return navigation.navigate('Profile');
     } else if (sheet === 'Remove') {
       const result = arrayImages.filter(item => item.id !== photoId);
       setArrayImage(result);
+      setModalVisible(false);
+    } else if (sheet === 'Cancel') {
+      setModalVisible(false);
     }
```
Comment (M7Arman) [inline screens/ImagesScreen/index.js:30]: you have setModalVisible(false); code in all 3 cases

Comment (MiqayelChaloyan) [reply screens/ImagesScreen/index.js:30]: Removed extra if body.

**Pull request merged**